setup.cfg

```diff
@@ -0,0 +1,27 @@
+[bumpversion]
+current_version = 0.1.0
+commit = True
+tag = True
+
+[bumpversion:file:setup.py]
+search = version="{current_version}"
+replace = version="{new_version}"
+
+[wheel]
+universal = 1
+
+[flake8]
+max-line-length = 120
```

Comment:
So it would just be too much to fix right now to go down to 80 characters, I guess?

Reply:
That was the idea 😅 
However, should we maybe review the choice of 80? On my workspace, 120 fits nicely in 2 vertically splitted editor windows, and would give some nice breathing room.